addons/website_blog/views/website_blog_templates.xml

```diff
@@ -257,13 +257,13 @@ list of filtered posts (by date or tag).
 ============================================================================ -->
 <template id="blog_post_content" name="Blog post content">
     <t t-if="opt_blog_post_breadcrumb and not opt_blog_post_regular_cover" t-call="website_blog.post_breadcrumbs">
-        <t t-set="additionnal_classes" t-value="'mb-0 pb-0 bg-transparent o_container_small pt-4'"></t>
+        <t t-set="additionnal_classes" t-value="'mb-0 pb-0 bg-transparent o_container_as_first pt-4'"></t>
     </t>
     <div t-field="blog_post.content"
         data-editor-message="WRITE HERE OR DRAG BUILDING BLOCKS"
         t-attf-class="o_wblog_post_content_field #{'js_tweet' if opt_blog_post_select_to_tweet else ''} #{'js_comment' if opt_blog_post_select_to_comment else ''} o_wblog_read_text"/>
 
-    <div t-if="len(blogs) > 1 or len(blog_post.tag_ids) > 0" class="css_editable_mode_hidden text-muted o_container_small">
+    <div t-if="len(blogs) > 1 or len(blog_post.tag_ids) > 0" class="text-muted o_wblog_post_tags o_container_as_first o_no_link_popover" contenteditable="false">
```

Comment:
`o_no_link_popover" contenteditable="false"` -> same comment as in the first commit: is this a fix? / o_not_editable?